reqs/test.pip

```diff
@@ -38,7 +38,7 @@ tensorflow==2.16.1; platform_machine != "arm64"
 keras==3.1.0; platform_machine != "arm64"
 
 # TensorFlow (arm64) related package. Currently no Python 3.13 support.
-tensorflow-macos==2.16.1; platform_machine == "arm64" and python_version < "3.13"
+tensorflow==2.16.1; platform_machine == "arm64" and python_version < "3.13"
```

Comment:
I don't think this makes sense. You probably want to remove this line and edit line 37 to covert both cases.